src/main/java/com/gluonhq/NativeBaseMojo.java

```diff
@@ -144,6 +153,11 @@ private void configSubstrate() {
                 break;
             case Constants.PROFILE_IOS:
                 targetTriplet = new Triplet(Constants.Profile.IOS);
+                IosSigningConfiguration signingConfiguration = new IosSigningConfiguration();
```

Comment:
Isn't it better to have this in Substrate? We now leak the IosSigningConfiguration to the plugin, which makes it much harder to modify that class. If we only send the Strings (identity/profile) to Substrate, this code block can be executed there.

Reply:
How would you send them? The only way will be adding these fields in the public `ProjectConfiguration`, which already has an exposed `IosSigningConfiguration` object.

Reply:
As Strings.

Reply:
I've created a follow-up issue for this